site/content/docs/terraform-provider/testing-scripts-locally.md

```diff
@@ -6,80 +6,203 @@ menu:
     parent: "Terraform provider"
 ---
 
-Having the possibility to run Playwright scripts for your browser checks locally will allow you to develop them faster and with more confidence.
+Having the possibility to run Playwright Test scripts for your browser checks locally will allow you to develop them faster and with more confidence.
 
 ## Enabling local script execution
 
 You can easily enable local execution by storing your browser check scripts in local folders and passing the path to the right script:
 
-```terraform
+{{< tabs "TF Check Resource" >}}
```

Comment:
@adamwardecki since the code is TF, I would ditch the double tab and just add a comment next to the `script` line saying something like "change to .js if you are using JS instead of Typescript"